src/main/java/winstone/realm/FileRealm.java

```diff
@@ -34,12 +34,12 @@
  */
 public class FileRealm extends HashLoginService {
     private static final WinstoneResourceBundle REALM_RESOURCES = new WinstoneResourceBundle("winstone.realm.LocalStrings");
-    
-    final String DEFAULT_FILE_NAME = "users.xml";
-    final String ELEM_USER = "user";
-    final String ATT_USERNAME = "username";
-    final String ATT_PASSWORD = "password";
-    final String ATT_ROLELIST = "roles";
+
+    static final String DEFAULT_FILE_NAME = "users.xml";
```

Comment:
Breaks public API?

Reply:
I don;t see how you should still be able to access the variable via a static or nonstatic way

Reply:
Right, sorry